openscap_daemon/cli_helpers.py

```diff
@@ -302,3 +302,27 @@ def summarize_standard_compliance_results(arf_source, result_list):
         result_json["Custom"] = {"XCCDF result": result}
 
         result_list.append(result_json)
+
+def get_list_of_profiles(sds_filepath):
```

Comment:
@mpreisler  Aha OK, I was looking for something like that before I implemented this, but I must be blind. I will try to reuse that here.